diff --git a/CMakeLists.txt b/CMakeLists.txt
index 796f963..25e88e2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.18)
 set (CMAKE_CXX_STANDARD 20)
-set (CMAKE_CXX_FLAGS "-Wno-unused-result -Wparentheses -Wsign-compare -DNDEBUG -g -O3 -Wall")
+set (CMAKE_CXX_FLAGS "-Wno-unused-result -Wparentheses -Wsign-compare -DNDEBUG -g -O3 -Wall -fPIC")
 
 project(openMTPK VERSION 0.5 LANGUAGES CXX)
 
@@ -39,7 +39,7 @@ set(SOURCE_FILES
     modules/nt/primes.cpp
 )
 
-add_library(${PROJECT_NAME} STATIC ${SOURCE_FILES})
+add_library(${PROJECT_NAME} SHARED ${SOURCE_FILES})
 
 #target_sources(${PROJECT_NAME} PUBLIC FILE_SET HEADERS
 #    BASE_DIRS ${OPENMTPK_SOURCE_DIR}
